core/object.h

```diff
@@ -124,10 +124,11 @@ enum PropertyUsageFlags {
 	PROPERTY_USAGE_NIL_IS_VARIANT = 1 << 19,
 	PROPERTY_USAGE_INTERNAL = 1 << 20,
 	PROPERTY_USAGE_DO_NOT_SHARE_ON_DUPLICATE = 1 << 21, // If the object is duplicated also this property will be duplicated
-	PROPERTY_USAGE_HIGH_END_GFX = 1 << 22,
-	PROPERTY_USAGE_NODE_PATH_FROM_SCENE_ROOT = 1 << 23,
-	PROPERTY_USAGE_RESOURCE_NOT_PERSISTENT = 1 << 24,
-	PROPERTY_USAGE_KEYING_INCREMENTS = 1 << 25, // Used in inspector to increment property when keyed in animation player
+	PROPERTY_USAGE_ALWAYS_SHARE_ON_DUPLICATE = 1 << 22, // If the object is duplicated this property will never be duplicated
```

Comment:
For 3.2 it probably should be added to the end of the enum, to avoid breaking compatibility with existing GDNative libraries.

And it should be added to the NativeScript enum in the https://github.com/godotengine/godot/blob/master/modules/gdnative/include/nativescript/godot_nativescript.h#L87 (relevant for both this and #41923).

Reply:
There is a lot of missing enums in NativeScript enum (PROPERTY_USAGE_SCRIPT_DEFAULT_VALUE and others are missed). I guess this should be added as a separate commit.